build.fsx

```diff
@@ -197,7 +503,119 @@ Target.create "ReleaseDocs" (fun _ ->
     Commit.exec tempDocsDir (sprintf "Update generated documentation for version %s" buildVersion)
     Branches.push tempDocsDir
 )
+*)
+
+let rec getUserInput (prompt : string) =
+    Console.Write $"%s{prompt}: "
+    let line = Console.ReadLine ()
+    if String.IsNullOrWhiteSpace line then
+        Console.WriteLine "Entry must be non-whitespace."
+        getUserInput prompt
+    else
+        line
+
+type GitHubAsset =
```

Comment:
Here follows some boilerplate to represent interacting with GitHub. See https://docs.github.com/en/rest/releases/releases?apiVersion=2022-11-28#create-a-release